recipes/srt/all/conanfile.py

```diff
@@ -52,6 +56,8 @@ def _configure_cmake(self):
         self._cmake.definitions["ENABLE_LOGGING"] = False
         self._cmake.definitions["ENABLE_SHARED"] = self.options.shared
         self._cmake.definitions["ENABLE_STATIC"] = not self.options.shared
+        if self._has_stdcxx_sync:
+            self._cmake.definitions["ENABLE_STDCXX_SYNC"] = True
```

Comment:
```suggestion        
        self._cmake.definitions["ENABLE_STDCXX_SYNC"] = self._has_stdcxx_sync
```

You can pass directly. If there no usage for older versions, CMake will only prompt a warning with other definitions.